grails-app/controllers/ygor/EnrichmentController.groovy

```diff
@@ -444,34 +444,18 @@ class EnrichmentController implements ControllersHelper{
           }
         }
       }
-      String tippNamespace = getTippNamespace(pkg)
+      def titleNamespace = getTitleNamespaceByPkgProvider(pkg)
+      String namespaceName = titleNamespace.name ?: titleNamespace.value ?: ""
```

Comment:
This would throw a NPE if no `titleNamespace` is set I think. -> `titleNamespace?.name`